modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -1245,6 +1248,42 @@ class DeConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
 
         pad.width = pad_l;
         pad.height = pad_t;
+
+        weightsMultipliers.assign(blobs[0].size[0], 1.0);
```

Comment:
Number of multipliers should be equal number of output channels (numOutput). However `blobs[0]` is a weights matrix with shape `IOHW` where `I` is a number of input channels and `O` is a number of output channels divided by group number. So it's better to use `numOutput` instead of raw shapes of weigths blob.